README.md

```diff
@@ -52,28 +52,28 @@ You should ideally provide a config file for your specific sensor, but default o
 
```

Comment:
[must] please add your cool description for nebula driver on top-level of this readme.

Reply:
Added in https://github.com/tier4/nebula/commit/7f5c0866d90470ee9e9a8ebcb568ec73002ebc95